bundles/http/Citrix/aa190bbb-79e8-4281-8bc4-7d7941432e9c/i18n/zh-CN.json

```diff
@@ -0,0 +1,77 @@
+{
+  "ef668059-dee5-4726-a8a9-acb8ae4ea708": {
+    "fb065fa6-9a9d-4177-889c-3d11aa43f2c0": "Timestamp 4 Writeback",
+    "3c5a9157-66e2-4bae-9dab-48fe4ea4cb3f": "新建",
+    "b59f7c64-7f51-4d56-9d82-8613f93d147a": "状态",
+    "ed75bb41-c6fc-4743-80a5-d5379290ae03": "类型",
+    "00895422-56d3-4a0c-8c93-cd0364f0ee59": "ID",
+    "c62606fa-04b6-479f-9f3c-e793b803b90f": "提交者",
+    "632c133e-89d0-4c23-b307-d09a48c70612": "说明",
+    "8cf3a7f9-424d-4c3d-bae9-af071aee18c3": "票证",
+    "aa85056b-1843-4127-9ca0-9cc82cf99938": "票证已更新",
+    "c760e7af-1d91-4c89-ab9f-e8e0ba7304ec": "票证 ID: {{id}} - {{subject}}",
+    "c526f82a-caae-4fe3-9c71-ab895c79af9e": "说明",
+    "9891eb45-5a84-4569-a278-eb90b0dfa899": "搜索票证",
+    "cd059fb2-7d9f-47fc-9963-caaae91f6cf5": "主题",
+    "0dd96f1f-d4d7-44ea-b396-dfcf8602b2d4": "Assigned to {{ticket_assignee_id_user.name}}",
```

Comment:
Unlocalized

Reply:
@yangwa-ctx this is a test PR. actual translations are in progress.